Trappist/src/Promact.Trappist.Web/wwwroot/app/questions/coding.language.enum.ts

```diff
@@ -0,0 +1,5 @@
+﻿export enum CodingLanguage {
```

Comment:
This is created to keep languages in order. 
The server will recognize 0 as 'Java', 1 as 'C++', 2 as 'C'. When the test conductor submits the question 
- the selected language will be of string type.
- the selected language may not be in order.
 Before calling post method the selected languages are converted into its respective number as defined in enum.